CollapseLauncher/Classes/Properties/AppActivation.cs

```diff
@@ -13,6 +15,33 @@ public static class AppActivation
         public static void Enable()
         {
             AppInstance.GetCurrent().Activated += App_Activated;
+
+            // Name for the protocol 
+            string name = "collapse";
```

Comment:
to avoid confusion later, while its just an internal thing, its better to name this something like `protocolName` or something like that rather than just a generic `name`

Reply:
Never noticed they were ordered alphabetically in other files
changed in c0a32a64